scripts/01_IAvH_test/area_layers.R

```diff
@@ -0,0 +1,72 @@
+# Instalar librerias necesarias

```

Comment:
Though I can read Spanish, I think it would be better for external audits to have these comments in English.